src/store/pageview.ts

```diff
@@ -3,9 +3,14 @@ import { PageView, StorageKeys, PageFeatureResult } from '../../types';
 
 class ViewStore {
   pageViews: PageView[];
+  storageSize: number;
 
-  constructor() {
+  /**
+   * @param storageSize Max pageView items to be kept
+   */
+  constructor(storageSize?: number) {
     this.pageViews = [];
+    this.storageSize = storageSize ?? Infinity;
```

Comment:
This is confusing as there is a default value, can we just set it here, and then allow the user to over write with the method.